README.md

```diff
@@ -20,8 +20,8 @@ To use the library directly, you need to use the `uring` ocamlfind library:
 Call `Uring.create` to initialise a ring:
 
 ```ocaml
-# let uring = Uring.create ~queue_depth:10 ();;
-val uring : '_weak1 Uring.t = <abstr>
+# let uring : [ `Open_log | `Close_log | `Write_all ] Uring.t = Uring.create ~queue_depth:10 ();;
+val uring : [ `Close_log | `Open_log | `Write_all ] Uring.t = <abstr>
 ```
 
 The `'_weak1` is the type of user data attached to requests, which can be whatever you want.
```

Comment:
For the README, it might be better to restrict it to OCaml 5.1 in the MDX. Then we can still show it doing inference.

Alternatively, the text needs changing so it no longer mentions `'_weak1`.

Reply:
Ah missed the comment, thanks! Forced pushed to use an `enabled_if` in the `dune` file 👍